src/ray/common/status.h

```diff
@@ -47,34 +48,35 @@ class error_code;
 }  // namespace boost
 
 // Return the given status if it is not OK.
-#define RAY_RETURN_NOT_OK(s)           \
-  do {                                 \
-    const ::ray::Status &_s = (s);     \
-    if (RAY_PREDICT_FALSE(!_s.ok())) { \
-      return _s;                       \
-    }                                  \
+#define RAY_RETURN_NOT_OK(s)                                \
+  do {                                                      \
+    const ::ray::Status &RAY_UNIQUE_VARIABLE(_s) = (s);     \
+    if (RAY_PREDICT_FALSE(!RAY_UNIQUE_VARIABLE(_s).ok())) { \
+      return RAY_UNIQUE_VARIABLE(_s);                       \
+    }                                                       \
   } while (0)
 
-#define RAY_RETURN_NOT_OK_ELSE(s, else_) \
-  do {                                   \
-    ::ray::Status _s = (s);              \
-    if (!_s.ok()) {                      \
-      else_;                             \
-      return _s;                         \
-    }                                    \
+#define RAY_RETURN_NOT_OK_ELSE(s, else_)                \
```

Comment:
Actually I think we can just delete this since it's dead code, no one uses it.

Reply:
OK, removed.